### DISCUSSION TITLE
[Request] Suggestions based on the c++ language version used

### DISCUSSION THREAD

**Discussion by mariocjun** [community/community#18099] [Actions]
Forgive me if I'm talking nonsense. While developing in C++98 (-std=c++98), I noticed that some suggestions were incompatible with the version I was working on. I would like to know if this is my fault in some way or flaw in the suggestions. Being the second option, at some point will this be resolved, improved or added?

**Comment by apetrai** [comment]
C++ has been standardised to C++20 now. This basically means that old C++ standards may not be supported any more. You should view your compilers documentation for more details.
